packages/amplify_auth_cognito/ios/Classes/FlutterSignUpRequest.swift

```diff
@@ -30,27 +30,18 @@ struct FlutterSignUpRequest {
     }
     
     private func createOptions(options: Dictionary<String, Any>?) -> AuthSignUpRequest.Options? {
-        if (options == nil) {
-            return nil
+        if ((options?["userAttributes"] is Dictionary<String, String>)) {
```

Comment:
Nit: Would usually prefer early return rather than wrapping most of the function in `if` clause. But if you do opt for this, I think you have an extra set of parens 😛

Reply:
```suggestion
        if (options?["userAttributes"] is Dictionary<String, String>) {
```